src/views/blocks/tickets/submit-button.php

```diff
@@ -23,14 +26,27 @@
  * @param string $button_name The button name. Set to cart-button to send to cart on submit, or set to checkout-button to send to checkout on submit.
  */
 $button_name = apply_filters( 'tribe_tickets_ticket_block_submit', 'cart-button' );
+
+/**
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters( 'tribe_tickets_ticket_block_submit_classes', [
+	'tribe-common-c-btn',
+	'tribe-common-c-btn--small',
+	'tribe-tickets__buy',
+] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: fc3e50cd104e6896a2335379d7cf9bdc</sub>